src/escalus_connection.erl

```diff
@@ -208,6 +209,11 @@ send(#client{module = Mod, event_client = EventClient, rcv_pid = Pid, jid = Jid}
     handle_stanza(Client, Elem, #{}, sent_stanza_handlers(Client)),
     ok.
 
+-spec send_raw(escalus:client(), iodata()) -> ok.
```

Comment:
```suggestion
-spec send_raw(escalus:client(), binary()) -> ok.
```

`iodata()` might be a list itself, but deeper in the call chain we also support `[exml_stream:element()]`. There's no guard allowing us to conveniently distinguish an `iodata()`/`iolist()` from an `[exml_stream:element()]`, while we only use `is_binary/1` in `escalus_tcp`. It's easiest and safest not to suggest `iodata()` here.